packages/kbn-test/src/functional_tests/cli/run_tests/args.js

```diff
@@ -93,6 +93,11 @@ export function processOptions(userOptions, defaultConfigPaths) {
     }
   }
 
+  if (userOptions['kibana-install-dir']) {
+    userOptions.installDir = userOptions['kibana-install-dir'];
+    delete userOptions['kibana-install-dir'];
+  }
+
```

Comment:
These lines need to be added to https://github.com/elastic/kibana/blob/master/packages/kbn-test/src/functional_tests/cli/start_servers/args.js#L82